src/sdk/staking.ts

```diff
@@ -1023,32 +1034,23 @@ export async function claim(
     const chainId = await getChainId(web3)
     const account = await getAccount(web3)
 
-    const simpleStakingAddresses = await getSimpleStakingContractAddresses(web3)
-    const simpleStakingAddressesv2 = await getSimpleStakingContractAddressesV2(web3)
+    const simpleStakingReleases = await getSimpleStakingContractAddressesV3(web3)
 
-    // todo: dry up
     const transactions: any[] = []
-    for (const address of simpleStakingAddresses) {
-        const [rewardG$, rewardGDAO] = await Promise.all([
-            getRewardG$(web3, address, account, false),
+    for (const releases of simpleStakingReleases) {
+      if (releases.release){
+        const isDeprecated = releases.release !== "v3"
+        for (const [key, address] of Object.entries(releases.addresses)){
+          const [rewardG$, rewardGDAO] = await Promise.all([
+            getRewardG$(web3, address, account, isDeprecated),
```

Comment:
i dont think isDeprecated is relevant, we do not claim from deprecated contracts